Remove incorrect randomness claim from privacy considerations (#482)
w3c/distributed-tracing: spec/50-privacy.md

## Privacy of traceparent field

> The `traceparent` field MUST NOT contain any personally identifiable information. One way to achieve this is to randomly generate all trace IDs. If a random number generator leverages any user identifiable information like IP address as seed state, this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user-identifiable.

Reviewer: ```suggestion
The `traceparent` field MUST NOT contain any personally identifiable information. One way to achieve this is to randomly generate all trace IDs using a random number generator that doesn't expose any personally identifiable information. For instance, if a random number generator uses IP address as seed state, it may expose this information. Hence, any random number generator used for generating trace IDs MUST NOT rely on any information that can potentially be personally identifiable.
```

Author: The use of the term "For instance" implies that you are showing an example of something, but the following sentence is not an example of the previous sentence. Reading the sentence again, I also think we don't need to provide IP as an example, because we don't want to define what might be personally identifiable. How about this?

```suggestion
The `traceparent` field MUST NOT contain any personally identifiable information. One way to achieve this is to randomly generate all trace IDs using a random number generator that does not expose any personally identifiable information. Any random number generator used for generating trace IDs MUST NOT rely on any information as input or seed state that can potentially be personally identifiable.
```

Reviewer: Thanks, this text looks good to me.
